dev-libs/libevent/libevent-2.1.11.ebuild

```diff
@@ -41,6 +41,7 @@ multilib_src_configure() {
 	ECONF_SOURCE="${S}" \
 	econf \
 		--disable-samples \
+		--enable-malloc-replacement \
 		$(use_enable debug debug-mode) \
 		$(use_enable debug malloc-replacement) \
```

Comment:
I guess you wanted to use --enable-malloc-replacement without setting USE=debug? Because that's what happens two lines below your insertion.

Reply:
it is enabled only in debug mode, I want to use it in release mode, sorry if I wrote in the wrong way.